CoreImportExportPlugins/test/unit/src/au/gov/asd/tac/constellation/plugins/importexport/ConfigurationPaneNGTest.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2010-2022 Australian Signals Directorate
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package au.gov.asd.tac.constellation.plugins.importexport;
+
+import au.gov.asd.tac.constellation.graph.Attribute;
+import au.gov.asd.tac.constellation.plugins.importexport.jdbc.JDBCImportController;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import javafx.scene.DepthTest;
+import javafx.scene.control.Tab;
+import org.mockito.Mockito;
+import org.testfx.api.FxToolkit;
+import static org.testng.Assert.*;
```

Comment:
are you able to import just the bits that are being used rather than the whole library?